lib/Onyx.js

```diff
@@ -801,22 +803,58 @@ function disconnect(connectionID, keyToRemoveFromEvictionBlocklist) {
     delete callbackToStateMapping[connectionID];
 }
 
+// [key, value, canUpdateSubscriber]
+let pendingUpdates = [];
+let scheduledPromise;
+let pendingCollectionUpdates = [];
```

Comment:
Could you also please add a doc comment for this (like you did for `pendingUpdates`)?